packages/did/src/Did.utils.ts

```diff
@@ -143,6 +152,34 @@ export function isSameSubject(
   return identifierA === identifierB
 }
 
+const signatureAlgForKeyType: Record<VerificationKeyType, string> = {
```

Comment:
C: Third time I am seeing a similar construct in the PR. Triggering my DRY receptors ;). Is it possible to optimize here?

Reply:
As we discussed https://github.com/KILTprotocol/sdk-js/pull/473/commits/bdeca25df838922d35d466449735a7c8cb010608